### ISSUE TITLE
Possible Issue with refine_actions

### ISSUE THREAD

**Issue by kwh5484** [facebookresearch/phyre#8]
In line 161 of neural_agent.py (in refine_actions), the cross entropy loss is calculated between the logits and a tensor of ones with length len(actions). Wouldn't it make more sense for the length to be len(action_batch)? In the present case, the code only runs without error if the values for the dqn-eval-batch-size and dqn-rank-size flags are equal.

**Comment by akhti** [comment]
Thanks, that's true. Not that we don't use action refining in any of the agents in the paper as we didn't see any clear improvement from doing that.
